docs/react/guides/cli.md

```diff
@@ -66,31 +66,270 @@ nx build myapp --prod
+| e2e       | Run all the E2E tests for the workspace                                                 |
+
+|
+| format:write | Run the code formatter across all the projects |
+| format:check | Check the formatted code across all the projects |
+| workspace-schematic | Run a workspace schematic |
+| test | Run the unit tests for all projects |
+| help | Display usage help for the Nx CLI |
+
+## Determining Affected Projects
+
+The Nx CLI also has built-in commands to show you how your changes impact applications and libraries across your workspace. The Nx CLI constructs and analyzes the dependency graph of all projects in your workspace, and determines the affected projects. These commands are helpful in isolating only the relevant applications, and libraries that were affected from a given change. They also allow you to streamline your CI processes, making them more efficient, only testing/building what's necessary, which improves CI turnaround time.
+
+> Each command can also be run using `nx affected --target={TARGET}`. They can also be provided `--base` and `--head` options that come from your Git repository.
+
+### affected:apps
+
+Display the list of apps affected by the current changes
+
+```sh
+nx affected:apps
+```
+
+---
+
+### affected:libs
+
+Display the list of libs affected by the current changes
+
+```sh
+nx affected:libs
+```
+
+---
+
+### affected:build
+
+Build the apps and libs affected by the current changes
+
+```sh
+nx affected:build
+```
+
+Affected builds can also be run in parallel
+
+```sh
+nx affected:build --parallel
+```
+
+After running the affected:build command, to only rerun failed projects:
+
+```sh
+nx affected:build --only-failed
+```
+
+You can also compare against a certain base and branch in CI
+
+```sh
+nx affected:apps --base=origin/master --head=$CI_BRANCH_NAME
+```
+
+---
+
+### affected:e2e
+
+Run the E2E tests for all projects affected by the current
+
+```sh
+nx affected:e2e
+```
+
+Link check for affected projects can also be run in parallel
+
+```sh
+nx affected:e2e --parallel
+```
+
+After running the affected:e2e command, to only rerun failed projects:
+
+```sh
+nx affected:e2e --only-failed
+```
+
+---
+
+### affected:test
+
+Run the unit tests for the projects affected by the current changes
+
+```sh
+nx affected:test
+```
+
+Link check for affected projects can also be run in parallel
+
+```sh
+nx affected:test --parallel
+```
+
+After running the affected:lint command, to only rerun failed projects:
+
+```sh
+nx affected:test --only-failed
+```
+
+---
+
+### affected:lint
+
+Run the lint check for the projects affected by the current changes
+
+```sh
+nx affected:lint
+```
+
+Link check for affected projects can also be run in parallel
+
+```sh
+nx affected:lint --parallel
+```
+
+After running the affected:lint command, to only rerun failed projects:
+
+```sh
+nx affected:lint --only-failed
+```
+
```

Comment:
Even though you mentioned it, maybe it's worth explicitly showing how to use nx affected --target=lint. Just so folks know that they can run anything with affected